src/components/App.js

```diff
@@ -62,7 +62,12 @@ export class App extends Component {
             authorization={this.props.authorization}
           />
           <Route path="/rejection" component={Rejection} />
-          <Route path="/confirmation" component={Confirmation} />
+          <Route
+            path="/confirmation"
+            component={Confirmation}
+            authentication={this.props.authentication}
```

Comment:
are these props meant to be passed into Confirmation?

Reply:
yes